config/src/test/resources/configmap-yml.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: app-config-yml
+data:
+  application.yml: |
+    hello:
+      message: configmap-yml
```

Comment:
You are using the same value for `message`, `epilogue` and `preamble` fields. Since you are checking the values of these fields later, it is better if they are different.

Reply:
Good point, they are prefixed with a field name now, thanks.